targets/targets.json

```diff
@@ -2092,113 +2042,6 @@
         "device_has_remove": ["QSPI"],
         "release_versions": ["2", "5"]
     },
-    "LPC55S69": {
```

Comment:
Why is that removed?

Reply:
This would obviously not be acceptable on `master`. This changeset is targeting a feature branch which aims only to work with PSoC 6. Removing LPC55S69 from targets.json was the most straightforward way to disable tests for targets we are deliberately breaking (and not yet fixing). When we take the branch back into `master`, we'll need to do the work to ensure this target works.